guides/common/modules/proc_configuring-project-server-as-red-hat-insights-client.adoc

```diff
@@ -13,23 +12,16 @@ For more information about Red{nbsp}Hat Insights, see https://access.redhat.com/
 
 To maintain your {ProjectServer}, and improve your ability to monitor and diagnose problems you might have with {Project}, install Red{nbsp}Hat Insights on {ProjectServer} and register {ProjectServer} with Red{nbsp}Hat Insights.
 
-.Scheduling insights-client
-
-Note that you can change the default schedule for running `insights-client` by configuring `insights-client.timer` on {Project}.
-For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
-
 .Procedure
-
-. To install Red{nbsp}Hat Insights on {ProjectServer}, enter the following command:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {project-package-install} insights-client
-----
-+
-. To register {ProjectServer} with Red{nbsp}Hat Insights, enter the following command:
+. Register {ProjectServer} with Red{nbsp}Hat Insights:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {foreman-installer} --register-with-insights
 ----
+For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/index#proc-registering-system-insights_insights-cg-configuring-client[Registering your system with Red Hat Insights] in the _Client Configuration Guide for Red Hat Insights_. 
+. Optional: You can change the default schedule for running `insights-client` by configuring `insights-client.timer` on {Project}.
+For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights/assembly-client-changing-schedule[Changing the insights-client schedule] in the _Client Configuration Guide for Red Hat Insights_.
+
```

Comment:
I'm wondering if we could include disabling the Insights client as a third optional step - it might be useful in upstream. Or is there a reason for removing it completely?

Reply:
Added back unregistering as an optional step. Since this section is about configuring, I don't think there's any issues with adding this in.